package.json

```diff
@@ -73,6 +73,10 @@
     "webpack": "1.13.3",
     "yargs": "8.0.1"
   },
+  "peerDependencies": {
+    "react": "^0.14.0 || ^15.0.0-0 || ^16.0.0-0",
+    "react-dom": "^0.14.0 || ^15.0.0-0 || ^16.0.0-0"
```

Comment:
I'm wondering what does `-0` mean at the end of each version.

Reply:
Play around with it [here](https://semver.npmjs.com/) by adding/removing the `-0`, but I think it allows versions such as `16.0.0-beta.1` to be included